### ISSUE TITLE
how to build baseline

### ISSUE THREAD

**Issue by kylewangt** [GACWR/OpenUBA#41]

**Comment by Jovonni** [comment]
Hello @kylewangt 

Can you provide some more details on the use case, or an example use case? We can then provide you with some steps, and we will cater the ongoing documentation branches to addressing this walkthrough as well. 

Since there are several approaches to baselining logs, let's start with these questions:

- What are you trying to baseline?
- What data sources are you using to baseline?

Lastly, if you can provide some thoughts on how you intend to use the baseline, we can assist with this in upcoming commits.

**Comment by anupamme** [comment]
Hello

If I am just trying to learn UBA is there a way to build an example baseline based on the data which is already present in the repository?

**Comment by Jovonni** [comment]
yes, we added a youtube video to help, and v2 shows how to do this more intuitively
